modules/features2d/misc/java/test/SURFDescriptorExtractorTest.java

```diff
@@ -29,7 +29,7 @@ protected void setUp() throws Exception {
         super.setUp();
 
         Class[] cParams = {double.class, int.class, int.class, boolean.class, boolean.class};
-        Object[] oValues = {100, 2, 4, true, false};
+        Object[] oValues = {100, 4, 2, true, false};
```

Comment:
Why?

Reply:
Ok, it was just to fit default constructor parameters, I keep old values (see final suggestion below)